### PR TITLE
[#156845313] add notification hub credentials

### PR DESCRIPTION
**Pull request by gunzip**
by now, we have no credentials for production

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Danger failed to run `Dangerfile.ts`.
  </td>
    </tr>
  </tbody>
</table>



## Error Error
```
Cannot find module 'danger-plugin-yarn'
Error: Cannot find module 'danger-plugin-yarn'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/circleci/repo/node_modules/danger-plugin-digitalcitizenship/dist/index.js:3:30)
    at Module._compile (module.js:570:32)
    at Object.customModuleHandler (/home/circleci/repo/node_modules/danger/distribution/runner/runners/inline.js:80:32)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```
### Dangerfile
```
------------------^
```
  
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: infrastructure/azure.tf [hunk 1]
```diff
@@ -143,11 +143,46 @@ variable "azurerm_apim_eventhub_rule" {
...
+  type        = "string"
+  description = "APNS name"
+}
+
+variable "notification_hub_apns_endpoint" {
+  type        = "string"
+  description = "APNS endpoint (test or sandbox)"
+}
+
+# This should be passed by ENV var NOTIFICATION_HUB_APNS_KEY
```
Comment (cloudify) [inline infrastructure/azure.tf]: `TF_VAR_NOTIFICATION_HUB_APNS_KEY`

File: infrastructure/azure.tf [hunk 2]
```diff
@@ -143,11 +143,46 @@ variable "azurerm_apim_eventhub_rule" {
...
+variable "NOTIFICATION_HUB_APNS_KEY" {
+  type        = "string"
+  description = "APNS Key"
+}
+
+# This should be passed by ENV var TF_VARS_NOTIFICATION_HUB_APNS_KEY_ID
```
Comment (cloudify) [inline infrastructure/azure.tf]: `TF_VAR_...`

File: infrastructure/azure.tf [hunk 3]
```diff
@@ -143,11 +143,46 @@ variable "azurerm_apim_eventhub_rule" {
...
+variable "NOTIFICATION_HUB_APNS_KEY_ID" {
+  type        = "string"
+  description = "APNS key Id"
+}
+
+# This should be passed by ENV var TF_VARS_NOTIFICATION_HUB_GCM_KEY
```
Comment (cloudify) [inline infrastructure/azure.tf]: `TF_VAR_...`

**Pull request merged**